docs/intro.md

```diff
@@ -4,4 +4,16 @@ sidebar_position: 1
 
 # Intro
 
-A hacker's guide to hackathons, made by the HackUTD Team.
```

Comment:
Nah we can keep this. But put it on the very first page of the tech guide website.

Reply:
Done.